cypress/integration/NoTrialsFound/DiseaseRedirectToNoTrialsFound.feature

```diff
@@ -1,40 +1,40 @@
 Feature: As the system, I want to be able to redirect users to the No Trials Found page for the dynamic listing pages, so that the pages are more user and SEO friendly.
 
-	Scenario: Page redirects to No Trials Found page if codes are given for Disease and no trials are returned
-		Given "trialListingPageType" is set to "Disease"
-		And "pageTitle" is set to "{{disease_label}} Clinical Trials"
-		And "noTrialsHtml" is set to "<p>There are no NCI-supported clinical trials for {{disease_normalized}} at this time. You can try a <a href=\"/about-cancer/treatment/clinical-trials/search\">new search</a> or <a href=\"/contact\">contact our Cancer Information Service</a> to talk about options for clinical trials.</p>"
-		Given the user navigates to "/C3037"
-		Then the user is redirected to "/chronic-fatigue-syndrome" with query parameters "redirect=true"
-		Then the user is redirected to "/notrials" with query parameters "p1=chronic-fatigue-syndrome"
-		Then the page title is "Chronic Fatigue Syndrome Clinical Trials"
-    And the system displays message "There are no NCI-supported clinical trials for chronic fatigue syndrome at this time. You can try a new search or contact our Cancer Information Service to talk about options for clinical trials."
+	Scenario: Page redirects to No Trials Found page if codes are given for Intervention and no trials are returned
```

Comment:
So it looks like the name of this test file and the following is other way around :  DiseaseRedirectToNoTrialsFound.feature is testing intervention and InterventionRedirectToNoTrialsFound.feature is for disease. Could you rename those two so they contain corresponding tests?